### PR TITLE
Add missing patch

### PR DESCRIPTION
**Pull request by denivyruck**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

The correct import should come from QtGui, but I left the previous one just in case.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,10 +9,11 @@ source:
   url: https://github.com/PierreRaybaut/guidata/archive/v{{ version }}.tar.gz
   sha256: 2ffc8e6dd27a57c6ec0e260986368beeddbd2fa5588f5200614b307b27a39e37
   patches:
-   - removed_entry_points.patch
+    - removed_entry_points.patch
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: Explanation why this patch is needed

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,10 +9,11 @@ source:
...
+    - qt_import.patch
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: ditto

File: recipe/meta.yaml (hunk 1 above)
Comment (denivyruck) [reply recipe/meta.yaml]: The first one I don't know the reason

File: recipe/meta.yaml (hunk 2 above)
Comment (denivyruck) [reply recipe/meta.yaml]: Done

File: recipe/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipe/meta.yaml]: If I do remember well, this patch is not necessary and you have to add the entry points in this recipe

Please see
https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#python-entry-points

Comment (denivyruck) [reply recipe/meta.yaml]: Ohhh cool

**Comment by marcelotrevisani** [comment]
It is not going to work, you need the migration of ``qt 4`` to the new compilers

**Comment by ocefpaf** [comment]
Done in #6

**Pull request closed without merging**